app/services/encrypted_email.rb

```diff
@@ -38,9 +36,11 @@ def fingerprint
   private
 
   attr_accessor :email, :encrypted_email
+  attr_writer :user_access_key
 
   def decrypt
     encryptor = Pii::PasswordEncryptor.new
-    encryptor.decrypt(encrypted_email, self.class.user_access_key)
+    self.user_access_key = self.class.user_access_key
```

Comment:
👃 This is a bit stinky. I'd like to see the class method named `build_user_access_key` again or something else meaningful (eg `user_access_key_from_env`?)